src/common/mathjax.js

```diff
@@ -53,7 +53,7 @@ goog.require('sre.System');
 
   (sre.System.getInstance()).setupEngine(
       {mode: sre.Engine.Mode.HTTP,
-       json: MathJax.Ajax.config.path['SRE'],
+       json: MathJax.Ajax.config.path['SRE'] + '/mathmaps/',
```

Comment:
It looks like `jsonPath` is used in two places, but inconsistently.

* [browser_tule.js line 105](https://github.com/zorkow/speech-rule-engine/blob/259d350ad02f47b8373c62c76f23f1e47aefc342/src/common/browser_util.js#L105) needs the extra `/` at the end, but

* [math_map.js line 172](https://github.com/zorkow/speech-rule-engine/blob/259d350ad02f47b8373c62c76f23f1e47aefc342/src/mathmaps/math_map.js#L172) adds it itself (giving you the double `//` in your example).